processor/src/main/java/org/mapstruct/ap/internal/util/Message.java

```diff
@@ -31,8 +31,8 @@
     BEANMAPPING_NO_ELEMENTS( "'nullValueMappingStrategy', 'resultType' and 'qualifiedBy' are undefined in @BeanMapping, define at least one of them." ),
     BEANMAPPING_NOT_ASSIGNABLE( "%s not assignable to: %s." ),
     BEANMAPPING_ABSTRACT( "The result type %s may not be an abstract class nor interface." ),
-    BEANMAPPING_UNKNOWN_PROPERTY_IN_RESULTTYPE( "Unknown property \"%s\" in return type. Did you mean \"%s\"?" ),
-    BEANMAPPING_PROPERTY_HAS_NO_WRITE_ACCESSOR_IN_RESULTTYPE( "Property \"%s\" has no write accessor." ),
+    BEANMAPPING_UNKNOWN_PROPERTY_IN_RESULTTYPE( "Unknown property \"%s\" in \"%s\". Did you mean \"%s\"?" ),
```

Comment:
How about `"Unknown property \"%s\" in result type %s. Did you mean \"%s\"?"` (adding “result type” again and removing the quotes around the type, as we usually only quote the property names... or do we?)